test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -16,6 +16,12 @@ namespace TorchSharp
 {
     public class TestTensor
     {
+        public TestTensor()
+        {
+            // Avoid comma issues on international machines (Swedish, for instance...)
+            CultureInfo.DefaultThreadCurrentCulture = (CultureInfo)CultureInfo.InvariantCulture.Clone();
```

Comment:
Good catch. 

I wonder if there's a need to force invariant culture on ToString() in general. I've been away from Sweden for so long, I'm afraid that I'm desensitized to number formatting, so I don't know what the right thing to do is.

Reply:
Lots of tests failed when running a swedish install ;)